exporter/azure_monitor/azure_monitor.go

```diff
@@ -3,60 +3,85 @@ package azure_monitor
 // This includes examples on how to create azure exporters to send spans.
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"log"
 
-	"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/exporter/azure_monitor/utils"
+	//"go.opencensus.io/exporter/azure_monitor/common"
 	"go.opencensus.io/trace"
 )
 
-type AzureTraceExporter struct {
-	InstrumentationKey string
-	Options            common.Options
+// Exporter is an implementation of trace.Exporter that uploads spans to Azure Monitor.
+type Exporter struct {
+	ServiceName         string
+	InstrumentationKey  string
+	Context             context.Context
+	EndPoint            string
+	TimeOut             int
 }
 
-/*	Creates an Azure Trace Exporter.
-	@param options holds specific attributes for the new exporter
-	@return The exporter created and error if there is any
-*/
-func NewAzureTraceExporter(IKey string) (*AzureTraceExporter, error) {
-	if IKey == "" {
+// Options are the options to be used when initializing an Azure Monitor exporter.
+type Options struct {

```

Comment:
Now we have `Exporter` and `Options` pretty much the same thing, why do we need both of them, or we just need one?

Reply:
I added the `Options` structure to follow the format of all the other exporters. The `NewExporter` function in each exporter accepts an `Options` parameter. A future implementation of this exporter may have more attributes in `Options` such as error handlers